rise/lib/resolvers/questions.ts

```diff
@@ -9,34 +8,41 @@ export const typeDefs = gql`
   }
 
   input Answer {
-    id: String!
     text: String!
+    id: String!
   }
 
   input Answers {
-    Answers: [Answer]
+    answers: [Answer]
+    user: String!
+  }
+
+  type latestAnswerResponse {
+    createdAt: String!
+    user: String!
   }
 
   extend type Mutation {
     sendAnswers(input: Answers!): Boolean!
+    latestAnswer(id: String!): latestAnswerResponse
   }
 
   extend type Query {
     questions: [Question!]!
   }
 `
 
-const sendAnswers: MutationResolvers['sendAnswers'] = (_parent, { input }) => {
-  console.log('sendAnswers: ', input)
-
-  return true
-}
 
 export const resolvers = {
   Query: {
     questions: getQuestions,
   },
   Mutation: {
-    sendAnswers,
+    sendAnswers: async (_ : any, { input: { answers, user } } : any) => {
```

Comment:
These can be typed, and inferred, see: 

https://github.com/Iteam1337/rise-backend/blob/master/rise/lib/resolvers/categories.ts
https://github.com/Iteam1337/rise-backend/blob/master/rise/lib/services/categories.ts#L5